package.json

```diff
@@ -134,20 +134,22 @@
     "polyfill-function-prototype-bind": "0.0.1",
     "quick_check": "^0.7.0",
     "raven-js": "^3.12.1",
-    "react": "^15.2.1",
+    "react": "^15.5.4",
     "react-addons-css-transition-group": "^15.3.2",
     "react-addons-perf": "^15.3.0",
     "react-addons-test-utils": "^15.3.0",
```

Comment:
Looks like this is [being depreciated](https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html#react-test-utils) in favor of `react-dom/test-utils`. We'll have to change...I think all our unit tests to import that instead.